pkg/infrastructure/mysql/place/repository_test.go

```diff
@@ -0,0 +1,292 @@
+package place
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"log"
+	"os"
+	"testing"
+	"time"
+	"wantum/pkg/domain/repository"
+	"wantum/pkg/domain/repository/place"
+	tx "wantum/pkg/infrastructure/mysql"
+	"wantum/pkg/infrastructure/mysql/model"
+
+	"github.com/stretchr/testify/assert"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+var db *sql.DB
+var txManager repository.MasterTxManager
+var repo place.Repository
+
+func TestMain(m *testing.M) {
+	before()
+	code := m.Run()
+	after()
+	os.Exit(code)
+}
+
+// repositoryを作ってもらう
+func before() {
+	var err error
+	// TODO: 環境変数とか使いたい気持ちもする
+	db, err = sql.Open("mysql", "root:root@tcp(localhost:3306)/wantum?parseTime=true")
+	if err != nil {
+		log.Fatal("faild to connect db: ", err)
+	}
```

Comment:
ここだけど、素直に`wantum/db/mysql`の`CreateSQLInstance`を使ってあげれば良さそう！
そうしてあげれば、DBの環境依存がなくせるかな！
```suggestion
	db := mysql.CreateSQLInstance()
```

Reply:
これなんやけど、テストで必ずローカルのDB使いたいから手動で毎回繋いでいるんよね。
`CreateSQLInstance`つかって流状態で、gcp導入しちゃうと、本番環境のDBでテスト始めてしまうかも知らんので、それを避けるために毎回作ってる感じです。。。
テスト用に接続用の関数作るのはありかも。。。

Reply:
@akubi0w1 
であれば、`mysql.connectLocalSQL()`をグローバル化してそれを呼び出したら良いんじゃない？？